src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -151,9 +151,9 @@ export class BaseEditCourseForm extends React.Component {
     this.setCollapsible(true);
   }
 
-  parseOptions(inChoices) {
-    return inChoices.map(choice =>
-      ({ label: choice.display_name, value: choice.value }));
+  parseOptions(choices, isLanguages) {
+    const options = choices.map(choice => ({ label: choice.display_name, value: choice.value }));
+    return isLanguages ? options.sort((a, b) => (a.label.localeCompare(b.label))) : options;
```

Comment:
I think this can be done more easily in the backend, sorting them when we retrieve the languages.

We should also avoid overloading this method with two different purposes. If we're going to keep the sorting in the frontend, we should consider creating a `parseLanguageOptions()` to handle it separately.